src/index.js

```diff
@@ -326,6 +329,22 @@ export default class AuthnWidget {
     }, 1000)
   }
 
+  postEmailVerificationRequired() {
+    clearTimeout(this.emailVerificationRequiredStateTimeout);
+    if (this.store.getStore().status !== this.store.getPreviousStore().status) {
+      if (document.querySelector("#notification")) {
```

Comment:
Can you elaborate on the scenario where the status would change between requests? I think that if the status changed, we would render a different template. I think this would only be needed if we were intercepting the form post and needed to process the response.

Reply:
Can you elaborate on the scenario where the status would change between requests?
A. This would happen when you login with an already registered user who has not verified their email. In this case, previous status is USERNAME_PASSWORD_REQUIRED and current status is EMAIL_VERIFICATION_REQUIRED.

Status would be remain the same if you are already in EMAIL_VERIFICATION_REQUIRED, and either do a resend poll, and the email is still not verified.

Reply:
I don't think there would be any notification shown since there wouldn't be a matching action in that scenario

Reply:
yes, you are right!

I can simply the logic to be:

```
  postEmailVerificationRequired() {
    if (this.store.getStore().status == this.store.getPreviousStore().status) {
      clearTimeout(this.emailVerificationRequiredStateTimeout);
      this.emailVerificationRequiredStateTimeout = setTimeout(() => {
        if (document.querySelector("#notification")) {
          document.querySelector('#notification').style.display = 'none';
        }
      }, 5000)
    }
  }
```

Reply:
Sounds good. I don't think the status check is necessary but is harmless. Thanks for changing